source/MRMesh/MRObjectPointsHolder.h

```diff
@@ -93,6 +93,23 @@ class MRMESH_CLASS ObjectPointsHolder : public VisualObject
         return renderDiscretization_;
     }
 
+    /// returns maximal number of points that will be rendered
+    /// if actual count of valid points is greater then the points will be sampled
+    int getMaxRenderingPoints() const
```

Comment:
mb rename to `getMaxAutoRenderingPoints` also please write in comment that this value is used only for auto discretization and user may set `getRenderDiscretization` manually